src/System.Runtime.CompilerServices.Unsafe/src/System.Runtime.CompilerServices.Unsafe.il

```diff
@@ -262,13 +262,19 @@
   .method public hidebysig static !!T& AsRef<T>(void* source) cil managed aggressiveinlining
   {
         .custom instance void System.Runtime.Versioning.NonVersionableAttribute::.ctor() = ( 01 00 00 00 )
+#ifdef NETCORE20
```

Comment:
Yeah but then I saw `CORE_ASSEMBLY` and the guide doesn't explicitly say anything about capitalization. But have fixed the two things.